README.md

```diff
@@ -7,3 +7,19 @@ Program for management and scheduling of Tasera managed shooting ranges in Pirka
 2. run npm install at /front
 3. run npm run dev at root
 4. The program should start up
+
+
+---- How to run the application (production) ----
+1. run npm install at root
+2. run npm install at /front
+3. run npm run build at root
+(1-3. alternative) use following in root:
+npm install && cd front && npm install && cd .. && npm run build 
```

Comment:
You could add a step here about how to run the dev version (npm run dev) or the production version (at server, need npm run build and others)

Reply:
eiks se oo tos selostetu jo